_topic_maps/_topic_map.yml

```diff
@@ -311,6 +311,14 @@ Topics:
     File: uninstalling-cluster-nutanix
   - Name: Installation configuration parameters for Nutanix
     File: installation-config-parameters-nutanix
+- Name: Installing on Oracle Cloud Infrastructure (OCI)
+  Dir: installing_oci
+  Distros: openshift-origin,openshift-enterprise
+  Topics:
+  - Name: Oracle Cloud Infrastructure (OCI) overview
+    File: installing-oracle-cloud-infra-overview
+  - Name: Using the Agent-based installer on OCI
```

Comment:
Hi @julienlim . This might be covered by another writer. AFAIA, certain installation routes will find their way into a knowledgebase article until the feature become tech preview of full-production features?